packages/@icon-magic/distribute/src/create-image-set.ts

```diff
@@ -29,7 +30,7 @@ interface AssetCatalog {
  */
 export async function createImageSet(iconSet: IconSet, outputPath: string) {
   for (const icon of iconSet.hash.values()) {
-    LOGGER.debug(`Creating imageSet for ${icon.iconName}`);
+    LOGGER.debug(`Creating imageSet for ${icon.iconName}, rtl: ${icon.rtlFlip}`);
```

Comment:
```suggestion
    LOGGER.debug(`Creating imageSet for ${icon.iconName}, rtl: ${icon.rtlFlip || false}`);
```

don't want it to log undefined